elastic/kibana
# [Ingest Pipelines] Highlight deprecated pipelines

Partially addresses https://github.com/elastic/kibana/issues/170805

## Summary

With https://github.com/elastic/elasticsearch/pull/101148 we now can alert the users when an ingest pipeline will soon be deprecated and shouldnt be relied upon. This PR adds a badge and a callout to the ingest pipelines UI to alert the users about this.

##### How to test

* Start up kibana and es
* Navigate to Stack management -> Ingest pipelines
* Verify that:
  * Deprecated pipelines are hidden by default
  * Upon enabling filtering to see them they should pop up on the table with a badge
  * Verify that deprecated policies are highlighted and when editing, we also show a warning callout.

### Screenshot 
<img width="1386" alt="Screenshot 2024-02-05 at 16 02 18" src="https://github.com/elastic/kibana/assets/6585477/739d6fea-5f15-4da4-aee8-cae8f412a040">
